.github/workflows/main.yml

```diff
@@ -31,3 +36,28 @@ jobs:
     
     - name: Publish Codecoverage report
       run: bash <(curl -s https://codecov.io/bash)
+
+    - name: Log in to the Container registry
+      uses: docker/login-action@v1
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Extract metadata (tags, labels) for Docker
+      id: meta
+      uses: docker/metadata-action@v3
+      with:
+        images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+        tags: |
+          type=ref,event=branch
+          type=ref,event=pr
+          type=semver,pattern={{version}}
+          type=semver,pattern={{major}}.{{minor}}
```

Comment:
Does not produce any latest tags anymore and also changes general style of our PR image names. Just stealing from metal-api or something alike makes it shorter, keeps existing tags and only requires a single workflow file.

Reply:
also copied over from api-server